cmd/cluster/create/runner.go

```diff
@@ -0,0 +1,57 @@
+package create
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kind/pkg/cluster"
+	"sigs.k8s.io/kind/pkg/cluster/config"
+	"sigs.k8s.io/kind/pkg/cluster/create"
+)
+
+const (
+	waitForReady = 2 * time.Minute
+)
+
+type runner struct {
+	flag   *flag
+	logger micrologger.Logger
+	stdout io.Writer
+	stderr io.Writer
+}
+
+func (r *runner) RunWithError(cmd *cobra.Command, args []string) error {
```

Comment:
Unrelated: let's rename that back to `Run`. `runner` doesn't fulfill any interface. It's ok to just return error in the signature.

Reply:
Good point. The `WithError` suffix isn't necessary. Done.